javascript-light-bulb/main.js

```diff
@@ -1,19 +1,21 @@
 /*
-- once the click occurs change the two class names.
-- once you click the button again it changes back to the original.
+-if the button is clicked change the class from light to dark.
+- if the button is clicked again change it from dark to light.
 */
+
 var $button = document.querySelector('.button-color');
 
 var $pageColor = document.querySelector('html');
 
-function lightSwitch(event){
-  if($button.className === 'button-color light' && $pageColor.className === 'html light'){
+var clicks = 0;
+
+$button.addEventListener('click', function (event){
+  clicks++;
+  if(clicks % 2 !== 0){
```

Comment:
A count of the clicks will work here, but you are keeping track of a count and just determining if that count is even or odd, one of two states. A boolean would be a better way to keep track of whether something is one of two different states. You should make a variable to keep track of if the light is currently on, with a boolean value.